### ISSUE TITLE
I can't set home... in my home

### ISSUE THREAD

**Issue by marcus47agent** [Luke100000/minecraft-comes-alive#248]
I don't know what happened, but I can't set home with any npc. The message that appears is: this place is not a construction. So I tried to create new maps and reproduce the same conditions as my house, or change it in the creative to see what happens and nothing happens, the same message repeats itself. I looked at the config files in the game folder, but not an option, or I didn't find it, that forces the npcs to set home in any location. Interestingly, any other menu option like "here is your work or leisure area" works
![2022-06-11_21 03 01](https://user-images.githubusercontent.com/107327385/173208948-94eb3eed-e102-4b3f-9d30-34b1fa536235.png)
![2022-06-11_21 02 24](https://user-images.githubusercontent.com/107327385/173208953-8c142c12-6adf-4ef4-81d9-a4bccd20e90c.png)

The outside of my house.

![2022-06-11_21 04 42](https://user-images.githubusercontent.com/107327385/173208974-96c05d9a-2dbb-428f-a923-fbf635252573.png)
![2022-06-11_21 04 46](https://user-images.githubusercontent.com/107327385/173208979-657f3171-c363-4627-9d47-ca55a7ed3736.png)
![2022-06-11_21 04 52](https://user-images.githubusercontent.com/107327385/173208985-f219f85b-7859-449f-ab22-43d04b6f5c01.png)
![2022-06-11_21 04 56](https://user-images.githubusercontent.com/107327385/173208990-4adcffcc-56ca-4b09-9f34-01ff4727932c.png)
![2022-06-11_21 05 02](https://user-images.githubusercontent.com/107327385/173208995-98567a79-30d1-418c-962b-b5c641422bcc.png)
![2022-06-11_21 05 06](https://user-images.githubusercontent.com/107327385/173208997-ee78a275-a81b-4009-9464-45ed9c9de351.png)

So I created a no world to test how the npcs home set really works. 
![2022-06-11_21 05 43](https://user-images.githubusercontent.com/107327385/173209023-4dd709a1-dbd5-45ff-b5f5-8b5498058a5f.png)
![2022-06-11_21 05 53](https://user-images.githubusercontent.com/107327385/173209048-1fb1486b-4a23-43ce-90fe-470e1665497f.png)
![Uploading 2022-06-11_21.05.53.png…]()
![2022-06-11_21 05 59](https://user-images.githubusercontent.com/107327385/173209050-da758d82-ba71-4f5a-afad-c74c7a9fb2c3.png)
![2022-06-11_21 06 05](https://user-images.githubusercontent.com/107327385/173209051-2b1b0a50-16e4-4996-91d5-c6ba6d672458.png)
![2022-06-11_21 06 34](https://user-images.githubusercontent.com/107327385/173209052-87aba0b5-e3ca-40c9-b874-863ba2595c95.png)

But a floating quartz cube can be set up like a house. I really can't understand how the sethome system was designed to work.

**Comment by marcus47agent** [comment]
I tried to reproduce the same conditions of the quartz cube in my original map and it didn't work, the problem remains
![2022-06-11_20 59 00](https://user-images.githubusercontent.com/107327385/173209078-5d0ed345-7f10-468b-9835-5ff79469a22f.png)
![2022-06-11_20 58 25](https://user-images.githubusercontent.com/107327385/173209083-b0a69d37-8407-494d-aa3b-ea2e0c740192.png)
![2022-06-11_20 58 39](https://user-images.githubusercontent.com/107327385/173209086-de6a22ec-46ae-49d2-af9c-30d929ed6137.png)
![2022-06-11_20 58 45](https://user-images.githubusercontent.com/107327385/173209087-b0c22947-f84c-4eca-bfe9-5bdd4a5e1084.png)
![2022-06-11_20 58 48](https://user-images.githubusercontent.com/107327385/173209088-6a66ffc6-ab25-45b2-8dbf-f631667ae5be.png)
![2022-06-11_20 58 53](https://user-images.githubusercontent.com/107327385/173209090-9da7a401-e40f-44e2-8d99-8e42308c78c3.png)
![2022-06-11_20 58 57](https://user-images.githubusercontent.com/107327385/173209092-9f9d8a0e-2e1b-48b6-93e6-55a45ae19e19.png)
.

**Comment by Luke100000** [comment]
Can you please use the blueprint to add the house? It will give you a better error message.

**Comment by marcus47agent** [comment]
> Can you please use the blueprint to add the house? It will give you a better error message.

Thanks in advance for your time.
Well, initially, I tried to do what you said, but I got an error that I needed to wait for some villager to accept a house that I had built. As I lived a fe
![2022-06-12_21 23 14](https://user-images.githubusercontent.com/107327385/173260083-ebf9c5d2-a86b-452a-a15c-2e4b8ef27287.png)
w steps from a village, I decided to use a map editor. I created a copy of my world and deleted the bridge and put my house literally glued to the village, so the blue print recognized the village already ready, so I clicked on add the structure, just like you said, and the blue print returned me a error that it's too big to be added, I don't know if the mod recognized only the house, or the walls and the entire area inside, which is a sq
![2022-06-12_21 31 27](https://user-images.githubusercontent.com/107327385/173260206-f048bccd-bc72-45b3-b621-3c64ab07edf0.png)
uare of 110 x 110 blocks
![2022-06-12_21 33 29](https://user-images.githubusercontent.com/107327385/173260289-9f3b1ac7-5831-4a0f-94ac-37c5f5f2ecda.png)
.
